lib/checkPullRequestBranch.js

```diff
@@ -23,28 +22,60 @@ const checkBranch = async (context) => {
    */
   const pullRequest = context.payload.pull_request;
   const branchName = pullRequest.head.ref;
-  const branchIsInvalid = DISSALLOWED_BRANCH_PREFIXES.some((prefix) => {
-    return branchName.startsWith(prefix);
-  });
 
-  if (branchIsInvalid) {
+  message = '';
+  invalid_branch_check = false;
+  
+
+  if(branchName === 'develop'){
+    
+    invalid_branch_check = true;
+    message = ', PRs made from develop branch are not allowed.'
+    
+  }
+
+  else if (branchName.startsWith('develop')) {
+
+    invalid_branch_check = true;
+    message = ', PRs made from a branch whose name is prefixed with develop are not allowed.';
+    
+  }
+
+  else if (branchName.startsWith('release')) {
+    
+    invalid_branch_check = true;
+    message = ', PRs made from a branch whose name is prefixed with release are not allowed.';
+              
+  }
+
+  else if (branchName.startsWith('test')) {
+
+    invalid_branch_check = true;
+    message = ', PRs made from a branch whose name is prefixed with test are not allowed.';
+              
+  }
```

Comment:
Here you can do this instead a shorten the code.
```suggestion
  else {
    invalid_prefixes = ['develop', 'release', 'test']
    for (i in invalid_prefixes) {
      if (branchName.startsWith(invalid_prefixes[i])) {
        invalid_branch_check = true;
        message = (
          `, PRs made from a branch whose name is prefixed with `
          `${invalid_prefixes[i]} are not allowed.`
        );            
      }
    }
```